packages/stream-model-instance-handler/src/schema-utils.ts

```diff
@@ -22,18 +19,17 @@ function buildAjv(): Ajv {
  */
 export class SchemaValidation {
   readonly ajv: Ajv
-  readonly schemas: LRUCache<string, ValidateFunction>
 
   constructor() {
     this.ajv = buildAjv()
-    this.schemas = new LRUCache(AJV_CACHE_SIZE)
   }
 
   public validateSchema(content: Record<string, any>, schema: SchemaObject, schemaId: string) {
-    let existingSchema = this.schemas.get(schemaId)
+    let existingSchema = this.ajv.getSchema(schemaId)
     if (!existingSchema) {
-      existingSchema = this.ajv.compile(schema)
-      this.schemas.set(schemaId, existingSchema)
+      this.ajv.addSchema(schema, schemaId)
+      //we've added the schema above, so ajv will have it
+      existingSchema = this.ajv.getSchema(schemaId)!
```

Comment:
is the exclamation point at the end intentional? If so what is it doing?

Reply:
Resolving the possible null value as not null. We just inserted, so it should never be null